cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/conformity/Cgmes3ConversionTest.java

```diff
@@ -70,30 +70,30 @@ public void microGridOperationalLimits() throws IOException {
         Network n = networkModel(Cgmes3Catalog.microGrid(), new Conversion.Config());
 
         Line ln = n.getLine("ffbabc27-1ccd-4fdc-b037-e341706c8d29");
-        assertEquals(1312.0, ln.getCurrentLimits1().getPermanentLimit(), 0.0);
-        assertEquals(1312.0, ln.getCurrentLimits2().getPermanentLimit(), 0.0);
+        assertEquals(1312.0, ln.getCurrentLimits1().map(LoadingLimits::getPermanentLimit).orElse(0.0), 0.0);
```

Comment:
Why do you have these 0 and -1 values?

Reply:
just for testing; for instance here we want to assert that there is a permanent current limit and that it is equal to 1312, so if there's no current limit we can put 0 so that the test fails